hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/buffer/chronicle/ChronicleMapMessageRepository.java

```diff
@@ -19,24 +20,63 @@
 
     private static final Logger logger = LoggerFactory.getLogger(ChronicleMapMessageRepository.class);
 
+    private static final boolean SAME_BUILDER_CONFIG = false;
+
     private ChronicleMap<String, ChronicleMapEntryValue> map;
 
+    public static ChronicleMapMessageRepository recover(File file) {
+        return new ChronicleMapMessageRepository(file);
+    }
+
+    public static ChronicleMapMessageRepository create(File file, int entries, int averageMessageSize) {
+        return new ChronicleMapMessageRepository(file, entries, averageMessageSize);
```

Comment:
Now, we have two constructors create() and recover() and I don't know which one to use. Can we do something with this?